homeassistant/components/airzone/climate.py

```diff
@@ -109,8 +109,11 @@ def __init__(
     ) -> None:
         """Initialize Airzone climate entity."""
         super().__init__(coordinator, entry, system_zone_id, zone_data)
+
+        _id = entry.entry_id if (entry.unique_id is None) else entry.unique_id
+
         self._attr_name = f"{zone_data[AZD_NAME]}"
-        self._attr_unique_id = f"{entry.entry_id}_{system_zone_id}"
+        self._attr_unique_id = f"{_id}_{system_zone_id}"
```

Comment:
We need to migrate the existing entities in the registry.

There is an example in `flux_led`

Reply:
Implemented in 74573f9b29c8b13b657e1173e913ccf9b5c81d38